### PR TITLE
Add fallbacks for performance timings

### PR DESCRIPTION
**Pull request by metal-messiah**
### Overview
This PR adds fallbacks to the old deprecated timings APIs for the edge of our supported browser list.

### CODE REVIEW

**Review by cwli24 — CHANGES_REQUESTED** [review]

**Comment by cwli24** [comment]
Does the unit test file need to include tests for the old API epoch values?

File: src/common/timing/nav-timing.js [hunk 1]
```diff
@@ -68,11 +68,20 @@ export function addPN (pn, v) {
...
-    val = offset ? Math.max(Math.round(value - offset), 0) : value
-    obj[prop] = Math.round(val)
+function handleValue (value, obj, prop, isOldApi) {
+  /*
+  For L2 Timing API, the value will already be a relative-to-previous-document DOMHighResTimeStamp.
+  For L1 (deprecated) Timing, the value is an UNIX epoch timestamp, which we will convert to a relative time using our offset.
+  //
+  PNT.type is reported as 0, 1, 2 -- 0 values must be honored to allow this
+  */
+  if (typeof value === 'number' && value >= 0) {
```
Comment (cwli24) [inline src/common/timing/nav-timing.js:78]: Backwards compat `value > 0` needed

File: src/common/timing/nav-timing.js [hunk 2]
```diff
@@ -34,30 +34,30 @@ const PNT_TYPES = {
   prerender: 3
 }
 
-export function addPT (offset, pt, v = {}, isLegacy) {
+export function addPT (offset, pt, v = {}, isL1Api = false) {
   if (!pt) return
   v.of = offset
   handleValue(0, v, 'n')
```
Comment (cwli24) [inline src/common/timing/nav-timing.js:40]: This `0` needs to be replaced with `v.of` when `value > 0` is enforced to keep 'n' in object.

Comment (metal-messiah) [reply src/common/timing/nav-timing.js:40]: good catch

File: src/common/timing/nav-timing.js [hunk 3]
```diff
@@ -27,43 +27,60 @@ var DOM_CONTENT_LOAD_EVENT = 'domContentLoadedEvent'
...
+  handleValue(pt.domComplete, v, 'dc', isL1Api)
+  handleValue(pt[LOAD_EVENT + START], v, 'l', isL1Api)
+  handleValue(pt[LOAD_EVENT + END], v, 'le', isL1Api)
   return v
 }
 
 // Add Performance Navigation values to the given object
 export function addPN (pn, v) {
-  handleValue(pn.type, v, 'ty')
+  handleValue(PNT_TYPES[pn.type], v, 'ty')
```
Comment (cwli24) [inline src/common/timing/nav-timing.js:66]: The L1 timing which have numeric `.type` would get undefined here instead of passing through

**Review by cwli24 — APPROVED** [review]

**Review by patrickhousley — APPROVED** [review]

**Pull request merged**